pkgs/development/python-modules/abjad/default.nix

```diff
@@ -1,13 +1,13 @@
-{
-  lib,
-  buildPythonPackage,
-  fetchPypi,
-  ply,
-  roman,
-  uqbar,
-  pythonOlder,
-  pytestCheckHook,
-  lilypond,
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ply
+, roman
+, uqbar
+, pythonOlder
+, pytestCheckHook
+, lilypond
+, typing-extensions
```

Comment:
Can you please remove this reformatting, it's unnecessary

Reply:
Even if it's specified that way in the [code conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax) section of the contributing page?

Reply:
Oh I didn't notice that bit, if you want to keep it please put the change in another commit. Also please change the PR title to something like `abjad: fix build`.

Reply:
All done!

Reply:
Could you please revert this change?
We have adopted `nixfmt-rfc-style` formatting for python modules. https://github.com/NixOS/nixpkgs/pull/313628

Reply:
@natsukium reverted!